From 0358bf8e34d1269c45953ddbc40009e4efef22d4 Mon Sep 17 00:00:00 2001
From: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Date: Thu, 2 Feb 2023 09:46:46 +0100
Subject: [PATCH 0153/1695] clk: stm32mp13: cosmetic: remove useless space

Remove prohibited space between 'static' and the macro 'STM32_GATE'

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 1b7ffcf0233a2a3296c1d1dd4b556d6a0747b252 clk: stm32mp13: cosmetic: remove useless space]
Change-Id: Iaac78f6db73a0e02f2840d009e8245e680690d88
Signed-off-by: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
---
 core/drivers/clk/clk-stm32mp13.c | 140 +++++++++++++++----------------
 1 file changed, 70 insertions(+), 70 deletions(-)

diff --git a/core/drivers/clk/clk-stm32mp13.c b/core/drivers/clk/clk-stm32mp13.c
index 994bba0f5..80e2a3647 100644
--- a/core/drivers/clk/clk-stm32mp13.c
+++ b/core/drivers/clk/clk-stm32mp13.c
@@ -2120,76 +2120,76 @@ static STM32_TIMER(ck_timg2, &ck_pclk2, 0, RCC_APB2DIVR, RCC_TIMG2PRER);
 static STM32_TIMER(ck_timg3, &ck_pclk6, 0, RCC_APB6DIVR, RCC_TIMG3PRER);
 
 /* Peripheral and Kernel Clocks */
-static  STM32_GATE(ck_ddrc1, &ck_axi, 0, GATE_DDRC1);
-static  STM32_GATE(ck_ddrc1lp, &ck_axi, 0, GATE_DDRC1LP);
-static  STM32_GATE(ck_ddrphyc, &ck_pll2r, 0, GATE_DDRPHYC);
-static  STM32_GATE(ck_ddrphyclp, &ck_pll2r, 0, GATE_DDRPHYCLP);
-static  STM32_GATE(ck_ddrcapb, &ck_pclk4, 0, GATE_DDRCAPB);
-static  STM32_GATE(ck_ddrcapblp, &ck_pclk4, 0, GATE_DDRCAPBLP);
-static  STM32_GATE(ck_axidcg, &ck_axi, 0, GATE_AXIDCG);
-static  STM32_GATE(ck_ddrphycapb, &ck_pclk4, 0, 0);
-static  STM32_GATE(ck_ddrphycapblp, &ck_pclk4, 0, GATE_DDRPHYCAPBLP);
-static  STM32_GATE(ck_syscfg, &ck_pclk3, 0, GATE_SYSCFG);
-static  STM32_GATE(ck_ddrperfm, &ck_pclk4, 0, GATE_DDRPERFM);
-static  STM32_GATE(ck_iwdg2, &ck_pclk4, 0, GATE_IWDG2APB);
-static  STM32_GATE(ck_rtcapb, &ck_pclk5, 0, GATE_RTCAPB);
-static  STM32_GATE(ck_tzc, &ck_pclk5, 0, GATE_TZC);
-static  STM32_GATE(ck_etzpcb, &ck_pclk5, 0, GATE_ETZPC);
-static  STM32_GATE(ck_iwdg1apb, &ck_pclk5, 0, GATE_IWDG1APB);
-static  STM32_GATE(ck_bsec, &ck_pclk5, 0, GATE_BSEC);
-static  STM32_GATE(ck_tim12_k, &ck_timg3, 0, GATE_TIM12);
-static  STM32_GATE(ck_tim15_k, &ck_timg3, 0, GATE_TIM15);
-static  STM32_GATE(ck_gpioa, &ck_mlahb, 0, GATE_GPIOA);
-static  STM32_GATE(ck_gpiob, &ck_mlahb, 0, GATE_GPIOB);
-static  STM32_GATE(ck_gpioc, &ck_mlahb, 0, GATE_GPIOC);
-static  STM32_GATE(ck_gpiod, &ck_mlahb, 0, GATE_GPIOD);
-static  STM32_GATE(ck_gpioe, &ck_mlahb, 0, GATE_GPIOE);
-static  STM32_GATE(ck_gpiof, &ck_mlahb, 0, GATE_GPIOF);
-static  STM32_GATE(ck_gpiog, &ck_mlahb, 0, GATE_GPIOG);
-static  STM32_GATE(ck_gpioh, &ck_mlahb, 0, GATE_GPIOH);
-static  STM32_GATE(ck_gpioi, &ck_mlahb, 0, GATE_GPIOI);
-static  STM32_GATE(ck_pka, &ck_axi, 0, GATE_PKA);
-static  STM32_GATE(ck_cryp1, &ck_pclk5, 0, GATE_CRYP1);
-static  STM32_GATE(ck_hash1, &ck_pclk5, 0, GATE_HASH1);
-static  STM32_GATE(ck_bkpsram, &ck_pclk5, 0, GATE_BKPSRAM);
-static  STM32_GATE(ck_dbg, &ck_axi, 0, GATE_DBGCK);
-static  STM32_GATE(ck_mce, &ck_axi, 0, GATE_MCE);
-static  STM32_GATE(ck_tim2_k, &ck_timg1, 0, GATE_TIM2);
-static  STM32_GATE(ck_tim3_k, &ck_timg1, 0, GATE_TIM3);
-static  STM32_GATE(ck_tim4_k, &ck_timg1, 0, GATE_TIM4);
-static  STM32_GATE(ck_tim5_k, &ck_timg1, 0, GATE_TIM5);
-static  STM32_GATE(ck_tim6_k, &ck_timg1, 0, GATE_TIM6);
-static  STM32_GATE(ck_tim7_k, &ck_timg1, 0, GATE_TIM7);
-static  STM32_GATE(ck_tim13_k, &ck_timg3, 0, GATE_TIM13);
-static  STM32_GATE(ck_tim14_k, &ck_timg3, 0, GATE_TIM14);
-static  STM32_GATE(ck_tim1_k, &ck_timg2, 0, GATE_TIM1);
-static  STM32_GATE(ck_tim8_k, &ck_timg2, 0, GATE_TIM8);
-static  STM32_GATE(ck_tim16_k, &ck_timg3, 0, GATE_TIM16);
-static  STM32_GATE(ck_tim17_k, &ck_timg3, 0, GATE_TIM17);
-static  STM32_GATE(ck_ltdc_px, &ck_pll4q, 0, GATE_LTDC);
-static  STM32_GATE(ck_dma1, &ck_mlahb, 0, GATE_DMA1);
-static  STM32_GATE(ck_dma2, &ck_mlahb, 0, GATE_DMA2);
-static  STM32_GATE(ck_adc1, &ck_mlahb, 0, GATE_ADC1);
-static  STM32_GATE(ck_adc2, &ck_mlahb, 0, GATE_ADC2);
-static  STM32_GATE(ck_mdma, &ck_axi, 0, GATE_MDMA);
-static  STM32_GATE(ck_eth1mac, &ck_axi, 0, GATE_ETH1MAC);
-static  STM32_GATE(ck_usbh, &ck_axi, 0, GATE_USBH);
-static  STM32_GATE(ck_vref, &ck_pclk3, 0, GATE_VREF);
-static  STM32_GATE(ck_tmpsens, &ck_pclk3, 0, GATE_DTS);
-static  STM32_GATE(ck_pmbctrl, &ck_pclk3, 0, GATE_PMBCTRL);
-static  STM32_GATE(ck_hdp, &ck_pclk3, 0, GATE_HDP);
-static  STM32_GATE(ck_stgenro, &ck_pclk4, 0, GATE_STGENRO);
-static  STM32_GATE(ck_dmamux1, &ck_axi, 0, GATE_DMAMUX1);
-static  STM32_GATE(ck_dmamux2, &ck_axi, 0, GATE_DMAMUX2);
-static  STM32_GATE(ck_dma3, &ck_axi, 0, GATE_DMA3);
-static  STM32_GATE(ck_tsc, &ck_axi, 0, GATE_TSC);
-static  STM32_GATE(ck_aximc, &ck_axi, 0, GATE_AXIMC);
-static  STM32_GATE(ck_crc1, &ck_axi, 0, GATE_CRC1);
-static  STM32_GATE(ck_eth1tx, &ck_axi, 0, GATE_ETH1TX);
-static  STM32_GATE(ck_eth1rx, &ck_axi, 0, GATE_ETH1RX);
-static  STM32_GATE(ck_eth2tx, &ck_axi, 0, GATE_ETH2TX);
-static  STM32_GATE(ck_eth2rx, &ck_axi, 0, GATE_ETH2RX);
-static  STM32_GATE(ck_eth2mac, &ck_axi, 0, GATE_ETH2MAC);
+static STM32_GATE(ck_ddrc1, &ck_axi, 0, GATE_DDRC1);
+static STM32_GATE(ck_ddrc1lp, &ck_axi, 0, GATE_DDRC1LP);
+static STM32_GATE(ck_ddrphyc, &ck_pll2r, 0, GATE_DDRPHYC);
+static STM32_GATE(ck_ddrphyclp, &ck_pll2r, 0, GATE_DDRPHYCLP);
+static STM32_GATE(ck_ddrcapb, &ck_pclk4, 0, GATE_DDRCAPB);
+static STM32_GATE(ck_ddrcapblp, &ck_pclk4, 0, GATE_DDRCAPBLP);
+static STM32_GATE(ck_axidcg, &ck_axi, 0, GATE_AXIDCG);
+static STM32_GATE(ck_ddrphycapb, &ck_pclk4, 0, 0);
+static STM32_GATE(ck_ddrphycapblp, &ck_pclk4, 0, GATE_DDRPHYCAPBLP);
+static STM32_GATE(ck_syscfg, &ck_pclk3, 0, GATE_SYSCFG);
+static STM32_GATE(ck_ddrperfm, &ck_pclk4, 0, GATE_DDRPERFM);
+static STM32_GATE(ck_iwdg2, &ck_pclk4, 0, GATE_IWDG2APB);
+static STM32_GATE(ck_rtcapb, &ck_pclk5, 0, GATE_RTCAPB);
+static STM32_GATE(ck_tzc, &ck_pclk5, 0, GATE_TZC);
+static STM32_GATE(ck_etzpcb, &ck_pclk5, 0, GATE_ETZPC);
+static STM32_GATE(ck_iwdg1apb, &ck_pclk5, 0, GATE_IWDG1APB);
+static STM32_GATE(ck_bsec, &ck_pclk5, 0, GATE_BSEC);
+static STM32_GATE(ck_tim12_k, &ck_timg3, 0, GATE_TIM12);
+static STM32_GATE(ck_tim15_k, &ck_timg3, 0, GATE_TIM15);
+static STM32_GATE(ck_gpioa, &ck_mlahb, 0, GATE_GPIOA);
+static STM32_GATE(ck_gpiob, &ck_mlahb, 0, GATE_GPIOB);
+static STM32_GATE(ck_gpioc, &ck_mlahb, 0, GATE_GPIOC);
+static STM32_GATE(ck_gpiod, &ck_mlahb, 0, GATE_GPIOD);
+static STM32_GATE(ck_gpioe, &ck_mlahb, 0, GATE_GPIOE);
+static STM32_GATE(ck_gpiof, &ck_mlahb, 0, GATE_GPIOF);
+static STM32_GATE(ck_gpiog, &ck_mlahb, 0, GATE_GPIOG);
+static STM32_GATE(ck_gpioh, &ck_mlahb, 0, GATE_GPIOH);
+static STM32_GATE(ck_gpioi, &ck_mlahb, 0, GATE_GPIOI);
+static STM32_GATE(ck_pka, &ck_axi, 0, GATE_PKA);
+static STM32_GATE(ck_cryp1, &ck_pclk5, 0, GATE_CRYP1);
+static STM32_GATE(ck_hash1, &ck_pclk5, 0, GATE_HASH1);
+static STM32_GATE(ck_bkpsram, &ck_pclk5, 0, GATE_BKPSRAM);
+static STM32_GATE(ck_dbg, &ck_axi, 0, GATE_DBGCK);
+static STM32_GATE(ck_mce, &ck_axi, 0, GATE_MCE);
+static STM32_GATE(ck_tim2_k, &ck_timg1, 0, GATE_TIM2);
+static STM32_GATE(ck_tim3_k, &ck_timg1, 0, GATE_TIM3);
+static STM32_GATE(ck_tim4_k, &ck_timg1, 0, GATE_TIM4);
+static STM32_GATE(ck_tim5_k, &ck_timg1, 0, GATE_TIM5);
+static STM32_GATE(ck_tim6_k, &ck_timg1, 0, GATE_TIM6);
+static STM32_GATE(ck_tim7_k, &ck_timg1, 0, GATE_TIM7);
+static STM32_GATE(ck_tim13_k, &ck_timg3, 0, GATE_TIM13);
+static STM32_GATE(ck_tim14_k, &ck_timg3, 0, GATE_TIM14);
+static STM32_GATE(ck_tim1_k, &ck_timg2, 0, GATE_TIM1);
+static STM32_GATE(ck_tim8_k, &ck_timg2, 0, GATE_TIM8);
+static STM32_GATE(ck_tim16_k, &ck_timg3, 0, GATE_TIM16);
+static STM32_GATE(ck_tim17_k, &ck_timg3, 0, GATE_TIM17);
+static STM32_GATE(ck_ltdc_px, &ck_pll4q, 0, GATE_LTDC);
+static STM32_GATE(ck_dma1, &ck_mlahb, 0, GATE_DMA1);
+static STM32_GATE(ck_dma2, &ck_mlahb, 0, GATE_DMA2);
+static STM32_GATE(ck_adc1, &ck_mlahb, 0, GATE_ADC1);
+static STM32_GATE(ck_adc2, &ck_mlahb, 0, GATE_ADC2);
+static STM32_GATE(ck_mdma, &ck_axi, 0, GATE_MDMA);
+static STM32_GATE(ck_eth1mac, &ck_axi, 0, GATE_ETH1MAC);
+static STM32_GATE(ck_usbh, &ck_axi, 0, GATE_USBH);
+static STM32_GATE(ck_vref, &ck_pclk3, 0, GATE_VREF);
+static STM32_GATE(ck_tmpsens, &ck_pclk3, 0, GATE_DTS);
+static STM32_GATE(ck_pmbctrl, &ck_pclk3, 0, GATE_PMBCTRL);
+static STM32_GATE(ck_hdp, &ck_pclk3, 0, GATE_HDP);
+static STM32_GATE(ck_stgenro, &ck_pclk4, 0, GATE_STGENRO);
+static STM32_GATE(ck_dmamux1, &ck_axi, 0, GATE_DMAMUX1);
+static STM32_GATE(ck_dmamux2, &ck_axi, 0, GATE_DMAMUX2);
+static STM32_GATE(ck_dma3, &ck_axi, 0, GATE_DMA3);
+static STM32_GATE(ck_tsc, &ck_axi, 0, GATE_TSC);
+static STM32_GATE(ck_aximc, &ck_axi, 0, GATE_AXIMC);
+static STM32_GATE(ck_crc1, &ck_axi, 0, GATE_CRC1);
+static STM32_GATE(ck_eth1tx, &ck_axi, 0, GATE_ETH1TX);
+static STM32_GATE(ck_eth1rx, &ck_axi, 0, GATE_ETH1RX);
+static STM32_GATE(ck_eth2tx, &ck_axi, 0, GATE_ETH2TX);
+static STM32_GATE(ck_eth2rx, &ck_axi, 0, GATE_ETH2RX);
+static STM32_GATE(ck_eth2mac, &ck_axi, 0, GATE_ETH2MAC);
 
 /* Kernel Clocks */
 static STM32_KCLK(ck_usbphy_k, 3,
-- 
2.47.2

